[BRD-125] - Monorepo (#3)
mia-platform/microlc: README.md

To install Yarn, run `npm install --global yarn`.

Once you have all the dependency in place, you can launch:

```shell
yarn install
yarn coverage
```

These two commands, will install the dependencies and run the tests with the coverage report that you can view as an
HTML page in `packages/*/coverage/lcov-report/index.html`.

### Run a package script

> This folder use yarn as package manager.

Reviewer: ```suggestion
This folder uses yarn as package manager.
```

> To run a script in a sub package, you should use `yarn workspace PACKAGE_NAME SCRIPT_NAME`.

Reviewer: ```suggestion
To run a script in a sub package, you can run `yarn workspace PACKAGE_NAME SCRIPT_NAME`.
```
